app/controllers/cards_controller.rb

```diff
@@ -1,5 +1,48 @@
 class CardsController < ApplicationController
+
+  before_action :find_card, only: [:update, :edit, :destroy]
+
   def index
     @card = Card.all
   end
+
+  def new
+    @card = Card.new
+  end
+
+  def create
+    @card = Card.new(card_params)
+
+    if @card.save
+      redirect_to cards_path
+    else
+      render 'new'
+    end
+  end
+
+  def update
+    if @card.update(card_params)
+      redirect_to cards_path
+    else
+      render 'edit'
+    end
+  end
+
+  def edit
+  end
+
+  def destroy
+    @card.destroy
+    redirect_to cards_path
+  end
+
+  def find_card
+    @card = Card.find(params[:id])
+  end
+
+private
```

Comment:
Indent access modifiers like `private`.<br>Keep a blank line before and after `private`.